website/content/guides/1.7-plugin-upgrade/index.mdx

```diff
@@ -0,0 +1,182 @@
+github.com/hashicorp/packer/common/net => github.com/hashicorp/packer-plugin-sdk/net
+github.com/hashicorp/packer/packer => github.com/hashicorp/packer-plugin-sdk/packer
+github.com/hashicorp/packer/builder => github.com/hashicorp/packer-plugin-sdk/packerbuilderdata
+github.com/hashicorp/packer/packer => github.com/hashicorp/packer-plugin-sdk/pathing
+github.com/hashicorp/packer/packer/plugin => github.com/hashicorp/packer-plugin-sdk/plugin
+github.com/hashicorp/packer/common/random => github.com/hashicorp/packer-plugin-sdk/random
+github.com/hashicorp/packer/common => github.com/hashicorp/packer-plugin-sdk/retry
+github.com/hashicorp/packer/packer/rpc => github.com/hashicorp/packer-plugin-sdk/rpc
+github.com/hashicorp/packer/communicator => github.com/hashicorp/packer-plugin-sdk/sdk-internals/communicator
+github.com/hashicorp/packer/common/shell => github.com/hashicorp/packer-plugin-sdk/shell
+github.com/hashicorp/packer/common/shell-local => github.com/hashicorp/packer-plugin-sdk/shell-local
+github.com/hashicorp/packer/helper/builder/localexec => github.com/hashicorp/packer-plugin-sdk/shell-local/localexec
+github.com/hashicorp/packer/common/shutdowncommand => github.com/hashicorp/packer-plugin-sdk/shutdowncommand
+github.com/hashicorp/packer/template => github.com/hashicorp/packer-plugin-sdk/template
+github.com/hashicorp/packer/packer/tmp => github.com/hashicorp/packer-plugin-sdk/tmp
+github.com/hashicorp/packer/helper/useragent => github.com/hashicorp/packer-plugin-sdk/useragent
+github.com/hashicorp/packer/common/uuid => github.com/hashicorp/packer-plugin-sdk/uuid
+**new** github.com/hashicorp/packer-plugin-sdk/version
+
+
+When in doubt you can search the packer-plugin-sdk repo for the name of your imported structs or functions to see where they exist in the sdk now; we have not changed struct or function names.
+
+## Upgrade plugins to use new multi-plugin rpc server
+
+### Why upgrade to the new rpc server?
+
+The goal of this server is to enable multiple related plugins, for example plugins that share a hypervisor or cloud, to rely on shared libraries without having to submodule common code. The helps maintainers who are experts in a specific technology to focus on that technology without having to maintain several repositories. Having a single global plugin wrapper also helps the Packer core register, download, and install plugins. Finally, it allows more clarity in the plugin binary naming and name-usage within a template.
+
+### Do I have to upgrade?
+
+For now, you do to have to upgrade your plugin. Packer will continue to understand how to discover and launch old-style plugins. However, you will not be able to register your plugin with HashiCorp until you have upgraded the plugin.
+
+### How to upgrade the plugin
+
+We've created a scaffolding repository over at https://github.com/hashicorp/packer-plugin-scaffolding/tree/main/builder/scaffolding
```

Comment:
Did you call out the builder scaffolding specifically to illustrate your example below? If so I made a small suggestion to the following text to call out the builder plugin directly. If this was indeed the intent I suggest changing this to

```suggestion
We've created a scaffolding repository over at https://github.com/hashicorp/packer-plugin-scaffolding/
```